packages/ui/constants.js

```diff
@@ -12,8 +12,21 @@ export const languages = [
   },
 ]
 
-export const CHART_UNITS = {
-  week: 'week',
-  month: 'month',
-  year: 'year',
+export const CHART_RANGE = {
+  d: {
+    key: 'd',
+    value: 24,
+  },
+  w: {
+    key: 'w',
+    value: 24 * 7,
+  },
+  m: {
+    key: 'm',
+    value: 24 * 30,
+  },
+  y: {
+    key: 'y',
+    value: 24 * 365,
+  },
```

Comment:
For the time being, I think we only need weekly and monthly ranges
```suggestion
export const CHART_RANGE = {
  w: {
    key: 'w',
    value: 24 * 7,
  },
  m: {
    key: 'm',
    value: 24 * 30,
  },
```